kahuna/public/js/components/gu-date/gu-date.js

```diff
@@ -0,0 +1,102 @@
+import angular from 'angular';
+import moment from 'moment';
+import Pikaday from 'pikaday';
+import 'pikaday/css/pikaday.css';
+
+import template from './gu-date.html';
+import './gu-date.css';
+
+const ISO8601_FORMAT = 'YYYY-MM-DDTHH:mm:ssZ';
+const DISPLAY_FORMAT = 'DD MMM YYYY';
+const TEN_YEARS_MILLIS = (10 * 365 * 24 * 60 * 60 * 1000);
```

Comment:
this seams a little simplistic (since years are a bit longer) but I assume you don't need this to be precise

Reply:
This is to limit the date selection in the UI. The variable name isn't that intention revealing however...

![image](https://user-images.githubusercontent.com/836140/44329577-bb3bb900-a45c-11e8-91ca-14f8e4608554.png)